Tella/Scenes/Uwazi/Views/Widgets/UwaziTextWidget.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  UwaziTextWidget.swift
+//  Tella
+//
+//  Created by Robert Shrestha on 9/11/23.
+//  Copyright © 2023 HORIZONTAL. All rights reserved.
+//
+
+import SwiftUI
+
+struct UwaziTextWidget: View {
+    @State var isValidText = false
+    @State var mandatoryError = false
+    @EnvironmentObject var prompt: UwaziEntryPrompt
+    @State var value: UwaziValue
+
+    var body: some View {
+        VStack(alignment: .leading) {
+                if mandatoryError {
+                    UwaziEntityMandatoryTextView()
+                }
+                TextfieldView(
+                    fieldContent: $value.stringValue,
+                    isValid: $isValidText,
+                    shouldShowError: .constant(false),
```

Comment:
should this be always false?

Reply:
According to the UI UX of this screen we are not using the common way of showing error i.e using shouldShowError rather than that we will be showing a text that show "This field is mandatory." below the title so it is set for as false permanently